oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/parser/listener/endpoint/MultiScopesSpanListener.java

```diff
@@ -161,7 +162,12 @@ public void parseEntry(SpanDecorator spanDecorator, SegmentCoreInfo segmentCoreI
             statement.setTraceId(traceId);
             for (KeyStringValuePair tag : spanDecorator.getAllTags()) {
                 if (SpanTags.DB_STATEMENT.equals(tag.getKey())) {
-                    statement.setStatement(tag.getValue());
+                    if (!StringUtil.isEmpty(tag.getValue()) && tag.getValue().length() > config.getMaxSlowSQLLength()) {
+                        statement.setStatement(tag.getValue().substring(0,config.getMaxSlowSQLLength()));
```

Comment:
Don't need to call getValue so many times. Provide a local variable named sqlStatement.